dependency-check-suppressions.xml

```diff
@@ -60,5 +60,15 @@
         <notes>HTTP Client 4.5.11 suppression, root cause and fix needs to be carried out separately</notes>
         <cve>CVE-2020-13956</cve>
     </suppress>
-    
+
+    <suppress>
+        <notes>This is because of the transitive dependency fasterxml:jackson for which mitigation is not yet available</notes>
+        <cve>CVE-2020-25649</cve>
```

Comment:
pretty sure there is mitigations, everyone else has upgraded it

Also never add an exclusion without an expiry date

Reply:
@timja As per the recommendations tomcat and Jackson databind jars are updated. Suppressions are also removed.